chaindexing-tests/src/index.ts

```diff
@@ -26,6 +26,41 @@ export class UnsavedContractAddressFactory {
   }
 }
 
+export class EventFactory {
+  static manyNew(size: number): Event[] {
+    return new Array(size).fill(null).map((_, index) => this.new(index));
+  }
+  static new(index = 0): Event {
+    return {
+      contractAddress: `0xBC4CA0EdA7647A8aB7C2061c2E118A18a93${index}f${getRandomInteger(
+        9
+      )}${getRandomInteger(9)}D`,
+      contractName: 'TestContract',
+      chainId: getRandomInteger(200),
+      abi: '[]',
+      logParams: {},
+      parameters: {},
+      topics: {},
+      blockHash: '0xabcdef123456789',
+      blockNumber: 123,
+      transactionHash: `0x789abcdef123456$${getRandomInteger(900)}`,
+      transactionIndex: 456,
+      logIndex: 789,
+      blockTimestamp: getRandomInteger(2000000),
+      removed: false
+    };
+  }
+
+  static isDeeplyEqual() {
+    const excludedField = ['insertedAt'];
+    return true;
+  }
```

Comment:
Where is this used?
```suggestion

```

Reply:
Seems I left that there by mistake. I have removed it now.